intelmq-manager/js/configs.js

```diff
@@ -704,11 +696,29 @@ function redrawNetwork() {
 function draw() {
     load_html_elements();
 
-    if (window.location.hash !== '#load') {
+    if (getUrlParameter("configuration") === "new") {
         app.nodes = {};
         app.edges = {};
     }
     initNetwork();
+    if (window.location.hash) {
```

Comment:
What is this block for? I'm just wondering because when appending `#foobar` or any other string to the URL triggers the error below.

Reply:
Hash now can contain only ID of an existing bot instance. If found, the network zooms to this bot and its node gets selected. :)

This is useful when clicking the gear icon next to the "Monitoring" header.